### PR TITLE
Update content from jira and LDM-533.tex

### PR DESCRIPTION
**Pull request by gcomoretto**

### CODE REVIEW

**Review by ebellm — APPROVED** [review]

File: jira_docugen.tex [hunk 1]
```diff
@@ -1,67 +1,55 @@
 \section{Test Cases Summary}\label{test-cases-summary}
 
+Follows the list of test cases documented in this specification.
```
Comment (jdswinbank) [inline jira_docugen.tex]: “The list of test cases documented in this test specification follows”.

But consider whether this text is necessary at all — I think it's self-evident what is being displayed.

File: jira_docugen.tex [hunk 2]
```diff
@@ -1,67 +1,55 @@
...
-\begin{longtable}[]{@{}llllll@{}}
+\begin{longtable}[]{llllll}
 \toprule
-Version & Status & Priority & Verification Type & Critical Event &
-Owner\tabularnewline
-\midrule
-\endhead
-1 & Approved & Normal & Test & False & Eric Bellm\tabularnewline
-\bottomrule
+Version & Status & Priority & Verification Type & Critical Event & Owner
```
Comment (jdswinbank) [inline jira_docugen.tex]: We're told that the “critical event” field is not used by DM — why are we including it here?

File: jira_docugen.tex [hunk 3]
```diff
@@ -74,55 +62,157 @@ \subsubsection{Test Items}\label{test-items}
...
+\subsubsection{Intercase Dependencies}
+None.
+
+
+\subsubsection{Environment Needs}
 
-- Input specification\\[2\baselineskip]No input data is required for
-this test case.
+\paragraph{Software}
+Software All prerequisite packages listed at
```
Comment (jdswinbank) [inline jira_docugen.tex]: Duplicate “Software”.

File: jira_docugen.tex [hunk 4]
```diff
@@ -74,55 +62,157 @@ \subsubsection{Test Items}\label{test-items}
...
+https://pipelines.lsst.io/install/ prereqs/centos.html must be available
```
Comment (jdswinbank) [inline jira_docugen.tex]: Erroneous space between `install/` and `prereqs`.

File: jira_docugen.tex [hunk 5]
```diff
@@ -74,55 +62,157 @@ \subsubsection{Test Items}\label{test-items}
...
+on the test system and on the LSST-VC compute node.
 
-\subsubsection{Test Script}\label{test-script}
 
-\textbf{Step 1}\\
-Release 16.0 of the LSST Science Pipelines will be installed into the
+\paragraph{Hardware}
+Hardware This test case shall be executed on a developer system at NCSA
```
Comment (jdswinbank) [inline jira_docugen.tex]: Duplicate “Hardware”.

File: jira_docugen.tex [hunk 6]
```diff
@@ -1,67 +1,55 @@
...
-\bottomrule
+Version & Status & Priority & Verification Type & Critical Event & Owner
+\\\midrule
+1 & Approved & Normal &
+Test & False & Eric Bellm
+\\\bottomrule
 \end{longtable}
 
-\subsubsection{Test Items}\label{test-items}
+\subsubsection{Requirements}
```
Comment (jdswinbank) [inline jira_docugen.tex]: These aren't requirements; they are verification elements.

File: jira_docugen.tex [hunk 7]
```diff
@@ -179,28 +286,28 @@ \subsubsection{Test Items}\label{test-items-1}
...
-  \href{https://jira.lsstcorp.org/browse/LVV-100}{LVV-100} -
-  DMS-REQ-0269-V-01: DIASource Catalog
-\item
-  \href{https://jira.lsstcorp.org/browse/LVV-102}{LVV-102} -
-  DMS-REQ-0271-V-01: DIAObject Catalog
-\end{itemize}
 
-\subsubsection{Precondition}\label{precondition-1}
+\subsubsection{Intercase Dependencies}
+LVV-T-17 (AG-00-00)
```
Comment (jdswinbank) [inline jira_docugen.tex]: But LVV-T-17 has no “intercase dependencies”! This doesn't seem consistent.

Can we adopt the (newer) “predecessors” nomenclature here for clarity?

File: jira_docugen.appendix.tex [hunk 8]
```diff
@@ -0,0 +1,99 @@
+\scriptsize{
```
Comment (jdswinbank) [inline jira_docugen.appendix.tex:9]: This is missing a `\section` heading.

File: jira_docugen.tex (hunk 3 above)
Comment (gcomoretto) [reply jira_docugen.tex]: Corrected in Jira.

File: jira_docugen.tex (hunk 4 above)
Comment (gcomoretto) [reply jira_docugen.tex]: Corrected in jira.

File: jira_docugen.tex (hunk 5 above)
Comment (gcomoretto) [reply jira_docugen.tex]: Corrected in Jira

File: jira_docugen.tex (hunk 6 above)
Comment (gcomoretto) [reply jira_docugen.tex]: OK, I will update the template in docsteady.

File: jira_docugen.tex (hunk 7 above)
Comment (gcomoretto) [reply jira_docugen.tex]: OK, I will update the template in docsteady and the confluence page.

File: jira_docugen.appendix.tex (hunk 8 above)
Comment (gcomoretto) [reply jira_docugen.appendix.tex:9]: OK, will update the templates in docsteady.

Comment (gcomoretto) [reply jira_docugen.appendix.tex]: The heading is in the master LDM-533.tex. No need to update the templates.

**Comment by gcomoretto** [comment]
The document has been update following @jdswinbank comments.

**Pull request merged**